Investigation/python/task_queue/async_file_task_demo/cluster_builder/hpa.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: frontend-scaler
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    name: worker-controller
+  minReplicas: 3
+  maxReplicas: 10
+  targetCPUUtilizationPercentage: 50
```

Comment:
Looks like it's a demo for just uploading, and the HPA will not be trigged in this demo, right?
If so, we'd better remove all HPA related snippets to avoid confusion.